app/controllers/goldencobra/api/v2/articles_controller.rb

```diff
@@ -34,47 +38,44 @@ def search
         # @return [json] Liefert Alle Artikel :id,:title, :ancestry
         # map{|c| [c.parent_path, c.id]}
         def index
-          require 'oj'
           if params[:article_ids].present?
             index_with_ids
           else
-            cache_key ||= ["indexarticles", Goldencobra::Article.all.pluck(:id, :ancestry, :title)]
-
-            @articles = Rails.cache.fetch(cache_key) do
-              Goldencobra::Article.select([:id, :title, :ancestry]).sort{ |a, b|
-                a[0] <=> b[0]
-              }
-            end
-
-            if params[:react_select] && params[:react_select] == "true"
-              # Die React Select Liste braucht das JSON in diesem Format. -hf
-              json_uploads = @articles.map{ |a| { "value" => a.id, "label" => a.parent_path } }
-            else
-              json_uploads = @articles.map{ |a| a.as_json(only: [:id, :title], methods: [:parent_path]) }
-            end
+            @articles = cached_articles
 
             respond_to do |format|
-              format.json { render json: Oj.dump({'articles' => json_uploads}, mode: :compat) }
+              format.json {
+                render json: Oj.dump({
+                  articles: articles_as_json
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.